bundle/language_en.properties

```diff
@@ -662,6 +664,52 @@ angal.hospital.visitduration.txt
 angal.hospital.visitendhour.txt                                                                        = Visit end hour
 angal.hospital.visitstarthour.txt                                                                      = Visit start hour
 angal.lab.allnegative.txt                                                                              = All negative.
+angal.inventory.acanceledinventorycannotbemodified.msg                                                 = A canceled inventory can not be modified
+angal.inventory.allproduct.radio                                                                       = All products
+angal.inventory.cannotcreateanotherinventorywithotherinprogressinthisward.msg                          = Cannot create another inventory with other in progress in this ward.
+angal.inventory.date.label                                                                             = Inventory's date
+angal.inventory.edit.title                                                                             = Edit an inventory
```

Comment:
Words in a dialog's title should start with a capital letter.   In this diff there is a mixture of capitalization.